socialforce/simulator.py

```diff
@@ -39,11 +41,11 @@ def __init__(self, initial_state, ped_space=None, delta_t=0.4, tau=0.5):
             self.state = np.concatenate((self.state, np.expand_dims(tau, -1)), axis=-1)
 
         # potentials
-        self.V = PedPedPotential(self.delta_t)
+        self.V = PedPedPotential(self.delta_t, v0, sigma)
         self.U = ped_space
 
         # field of view
-        self.w = FieldOfView()
```

Comment:
similarly here `self.w = field_of_view or FieldOfView()`

Reply:
Can you please add `or FieldOfView()` so that default behavior is restored?
Similarly for `ped_ped` above.